sky/packages/sky/lib/sky_tool

```diff
@@ -179,15 +198,26 @@ class StartSky(object):
                               "exist to locate %s." \
                               % (os.path.basename(__file__), APK_NAME))
                 return 2
-            if args.build_path is not None:
-                apk_path = os.path.join(args.build_path, 'apks', APK_NAME)
+            if args.local_build:
+                apk_path = os.path.join(os.path.normpath(args.sky_src_path), args.android_debug_build_path, 'apks', APK_NAME)
             else:
                 apk_path = os.path.join(APK_DIR, APK_NAME)
             if not os.path.exists(apk_path):
                 logging.error("'%s' does not exist?" % apk_path)
                 return 2
 
-            subprocess.check_call([ADB_PATH, 'install', '-r', apk_path])
+            cmd = [ADB_PATH, 'install', '-r', apk_path]
+            subprocess.check_call(cmd)
```

Comment:
I'm planning on adding a verbose mode that logs all of the commands that get executed.  I already have it locally, but not in a well-factored state, so there are a number of lines like this, but they will be refactored soon.